css/properties/margin-block-end.json

```
@@ -6,15 +6,51 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/margin-block-end",
           "spec_url": "https://drafts.csswg.org/css-scroll-snap/#scroll-padding",
           "support": {
-            "chrome": {
-              "version_added": "69"
-            },
-            "chrome_android": {
-              "version_added": "69"
-            },
-            "edge": {
-              "version_added": "79"
-            },
+            "chrome": [
+              {
+                "version_added": "87"
+              },
+              {
+                "version_added": "69",
```

Presumably you can't turn this off anymore (there's no retroactive option for experimental web platform features). This needs to be corrected for this entry and any other case where a feature graduated from experimental web platform features.

```suggestion
                "version_added": "69",
                "version_removed": "87",
```